src/org/ods/orchestration/util/MROPipelineUtil.groovy

```diff
@@ -296,90 +299,74 @@ class MROPipelineUtil extends PipelineUtil {
             "*/${branch}",
             [
                 [ $class: 'RelativeTargetDirectory', relativeTargetDir: "${REPOS_BASE_DIR}/${repo.id}" ],
-                [ $class: 'LocalBranch', localBranch: "**" ]
+                [ $class: 'LocalBranch', localBranch: "**" ],
             ],
             [[ credentialsId: credentialsId, url: repo.url ]]
         )
     }
 
-    Set<Closure> prepareExecutePhaseForRepoNamedJob(String name, Map repo, Closure preExecute = null, Closure postExecute = null) {
```

Comment:
This monster is the problem that this PR addresses. Instead of having this huge, complicated block here in utils, it is now easy to read from each stage.